Sources/Sentry/SentryBreadcrumbStore.m

```diff
@@ -25,41 +25,53 @@
 @interface SentryBreadcrumbStore ()
```

Comment:
You could actually get rid of this class and add those methods directly in the scope.
You would only need 
`@property(nonatomic, strong) NSMutableArray<SentryBreadcrumb *> *breadcrumbs;`

The rest could be inlined in the scope functions.

Reply:
already did that, but inside `SentryBreadcrumbStore`. IMHO this is good since breadcrumbs need to be serialized. Like to keep this in one class. makes it also easier to change this later. cleaned it up and renamed it to `SentryBreadcrumbs`.

@HazAT should I drop this and move everything into the Scope? is the naming ok? `SentryBreadcrumbManager` is the only alternative that comes to my mind.

Reply:
So since we need the serialization in the scope anyway for simplicity I would move everything into the scope.

Reply:
done.